docs/testing.md

```diff
@@ -2,19 +2,37 @@
 
 ## Quick start:
 
-Many of our browser tests have been rewritten as Python unit tests. The remaining rely on data that is not generated during testing, so please note many of these tests might fail if you do not have a production database locally. If you do, open a new Terminal window or tab and change to the project directory,
+To run browser tests, open a new Terminal window or tab and change to the project directory,
 then tell gulp to start the tests:
 
 ```sh
 gulp build
-gulp test:acceptance --suite=integration
-gulp test:acceptance --suite=content
+gulp test:acceptance
+tox -e acceptance
```

Comment:
ok, I added the language in https://github.com/cfpb/cfgov-refresh/pull/3012/commits/3224e84b884fa360d7cd132eec5f8b56cc7a0873. @anselmbradford, if have specific language in mind jump in and do your thing.